app/src/main/res/layout/add_profile_activity.xml

```diff
@@ -17,144 +17,111 @@
     android:layout_height="match_parent">
 
     <ScrollView
-      android:id="@+id/scroll"
+      android:id="@+id/add_profile_activity_scroll_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
       android:background="@color/addProfileBackground">
 
       <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:paddingTop="40dp"
-        android:paddingBottom="40dp">
-
-        <de.hdodenhof.circleimageview.CircleImageView
-          android:id="@+id/upload_image_button"
-          android:layout_width="136dp"
-          android:layout_height="136dp"
-          android:paddingBottom="8dp"
+        android:paddingTop="48dp"
```

Comment:
These are the screenshots across various devices. I do understand that this is the same issue that we faced in ProfileProgressActivity so better link this to #980 

Pixel 3A XL - API 28.
<img width="207" alt="Screenshot 2020-04-20 at 12 52 43 PM" src="https://user-images.githubusercontent.com/9396084/79725173-75588780-8306-11ea-8965-2792a9ec9dfd.png">

Galaxy Nexus - 28
<img width="161" alt="Screenshot 2020-04-20 at 12 53 40 PM" src="https://user-images.githubusercontent.com/9396084/79725179-77bae180-8306-11ea-82b7-c33ec84a4e5c.png">

Nexus S Screenshot - 29
<img width="181" alt="Screenshot 2020-04-20 at 12 54 26 PM" src="https://user-images.githubusercontent.com/9396084/79725182-78ec0e80-8306-11ea-8d3e-60496c565973.png">

Reply:
Yes, Done